tests/functional/test_contract_instance.py

```diff
@@ -38,9 +38,27 @@ def test_init_specify_contract_type(
 def test_contract_calls(owner, contract_instance):
     contract_instance.setNumber(2, sender=owner)
     assert contract_instance.myNumber() == 2
-    contract_instance.invoke_transaction("setNumber", 3, sender=owner)
+
+
+def test_invoke_transaction(owner, contract_instance):
+    # Test mutable method call with invoke_transaction
+    receipt = contract_instance.invoke_transaction("setNumber", 3, sender=owner)
+    print(receipt)
```

Comment:
```suggestion
```

Reply:
@unparalleled-js implemented in the last commit 😄